docs/src/main/asciidoc/_cloud/References.adoc

```diff
@@ -0,0 +1,21 @@
+# References
+
+# Helm
+* https://helm.sh/[Official page]
+* https://helm.sh/docs/intro/install/[Installing Helm]
+* https://github.com/wildfly/wildfly-charts/blob/main/charts/wildfly/README.md[Helm Chart for WildFly]
+** https://www.wildfly.org/news/2021/05/05/helm-charts-for-wildfly/[Related blog post]
+* https://docs.wildfly.org/wildfly-charts/[Helm Charts for WildFly]
+
+# OpenShift
+* https://access.redhat.com/downloads/content/290/ver=4.12/rhel---9/4.12/x86_64/product-software[Download OpenShift]
+* https://developers.redhat.com/products/openshift-local/overview[Red Hat OpenShift Local]
+* https://www.redhat.com/en/technologies/cloud-computing/openshift/try-it[Try Red Hat OpenShift]
+* https://docs.okd.io/4.11/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli_cli-developer-commands[Getting started with the OpenShift CLI]
+
+# WildFly
+* https://docs.wildfly.org/wildfly-maven-plugin/[WildFly Maven Plugin (wildfly-maven-plugin)]
+* https://github.com/wildfly-extras/wildfly-cloud-galleon-pack/blob/main/doc/index.md[WildFly Cloud Galleon Pack’s documentation]
```

Comment:
@jmfinelli , the todo backend depends on the wildfly-datasources-galleon-pack (https://github.com/wildfly-extras/wildfly-datasources-galleon-pack/). It would be nice to add a link to it. Documentation for it is accessible from the README.